Library/Formula/zssh.rb

```diff
@@ -15,4 +13,8 @@ def install
     bin.install "zssh", "ztelnet"
     man1.install "zssh.1", "ztelnet.1"
   end
+
+  test do
+    system bin/"zssh", "-V"
```

Comment:
Failing on the bots: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/22960/version=yosemite/testReport/junit/brew-test-bot/yosemite/test_zssh/

Might be because there isn't a shell? Perhaps you can set `TERM` or something to work around this.